gtsfm/frontend/verifier/ransac.py

```diff
@@ -11,122 +11,84 @@
 Authors: John Lambert
 """
 
-from typing import Optional, Tuple
+from typing import Tuple
 
 import cv2
 import numpy as np
-from gtsam import Cal3Bundler, Rot3, Unit3
 
-import gtsfm.utils.features as feature_utils
 import gtsfm.utils.logger as logger_utils
-import gtsfm.utils.verification as verification_utils
 from gtsfm.common.keypoints import Keypoints
-from gtsfm.frontend.verifier.verifier_base import VerifierBase, NUM_MATCHES_REQ_E_MATRIX, NUM_MATCHES_REQ_F_MATRIX
+from gtsfm.frontend.verifier.opencv_verifier_base import OpencvVerifierBase
 
 
 RANSAC_SUCCESS_PROB = 0.999999
 RANSAC_MAX_ITERS = 1000000
 
 logger = logger_utils.get_logger()
 
+"""Alternative options:
```

Comment:
These are options just for E matrix, right? Please add options for the F matrix too

Reply:
Added more options as an Enum.